joml-geometry/src/main/java/org/terasology/joml/geom/AABBd.java

```diff
@@ -281,6 +301,62 @@ public Vector3f center(Vector3f dest) {
         return dest.set(minX + ((maxX - minX) / 2.0f), minY + ((maxY - minY) / 2.0f), minZ + ((maxZ - minZ) / 2.0f));
     }
 
+    @Override
+    public AABBd expand(double dx, double dy, double dz, AABBd dest) {
+        return dest.set(
+            minX - dx,
+            minY - dy,
+            minZ - dz,
+            maxX + dx,
+            maxY + dy,
+            maxZ + dz
+        );
+    }
+
+    /**
+     *
+     *
+     * @param dx the x component
+     * @param dy the y component
+     * @param dz the z component
+     * @return this
+     */
+    public AABBd expand(double dx, double dy, double dz) {
+        return expand(dx, dy, dz, this);
+    }
+
+    @Override
+    public AABBd expand(Vector3fc extent, AABBd dest) {
+        return dest.set(
+            minX -  extent.x(),
+            minY -  extent.y(),
+            minZ -  extent.z(),
+            maxX +  extent.x(),
+            maxY +  extent.y(),
+            maxZ +  extent.z()
+        );
```

Comment:
I'd rather avoid implementing the same thing multiple times and instead rely on the JIT compiler to optimize the additional method call away (I think this will typically be the case).

```suggestion
        return expand(extent.x(), extent.y(), extent.z(), dest);
```